content/chapitres/ci.adoc

```diff
@@ -164,25 +219,25 @@ include::../code-samples/gh-actions/say-hello-full.yml[tags="common,checkout,sho
 
 == Exemple : Environnement d'exécution
 
-* Notre pipeline de build dit que "la vache" doit afficher le contenu du fichier `README.md`
-** WAT (Non, nous ne sommes pas fous) ?
+* Notre pipeline de build doit s'assurer que "la vache" 🐮 doit nous lire 💬 le contenu du fichier `README.md`
```

Comment:
```suggestion
* Notre workflow doit s'assurer que "la vache" 🐮 doit nous lire 💬 le contenu du fichier `README.md`
```

Terminology nit: pipeline doesn't mean much in the github action universe: we should stick to workflow?